ArmPkg/ArmPkg.dec

```diff
@@ -41,10 +41,6 @@
   #
   ArmHvcLib|Include/Library/ArmHvcLib.h
 
-  ##  @libraryclass  Provides a Mmu interface.
-  #
-  ArmMmuLib|Include/Library/ArmMmuLib.h
-

```

Comment:
This library class could live in UefiCpuPkg too, no?

Reply:
yes, it has been moved to UefiCpuPkg in this PR.